CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.11.8
+- chore: Add peers to specific defined sessions instead of sending the entire wantlist. [PR XXX](https://github.com/dariusc93/rust-ipfs/pull/XXX)
```

Comment:
```suggestion
- chore: Add peers to specific defined sessions instead of sending the entire wantlist. [PR 189](https://github.com/dariusc93/rust-ipfs/pull/189)
```